package.json

```diff
@@ -26,24 +30,23 @@
 		"url": "https://github.com/amwmedia/plop/issues"
 	},
 	"devDependencies": {
+		"chalk": "^4.1.1",
 		"inquirer-directory": "^2.2.0",
-		"plop-pack-fancy-comments": "^0.2.0",
-		"webpack": "^4.43.0",
-		"webpack-cli": "^3.3.11",
-		"chalk": "^1.1.3",
-		"interpret": "^1.2.0",
-		"liftoff": "^2.5.0",
-		"minimist": "^1.2.0",
-		"node-plop": "~0.25.0",
-		"ora": "^3.4.0",
-		"string-replace-loader": "^2.3.0",
-		"v8flags": "^2.0.10"
+		"interpret": "^2.2.0",
+		"liftoff": "^3.1.0",
+		"minimist": "^1.2.5",
+		"node-plop": "~0.26.2",
+		"ora": "^5.4.0",
+		"string-replace-loader": "^3.0.1",
+		"v8flags": "^3.2.0",
+		"webpack-cli": "^4.7.0",
+		"webpack": "^5.37.0"
 	},
 	"homepage": "https://plopjs.com",
 	"engines": {
 		"node": ">=8.9.4"
 	},
-	"preferGlobal": "true",
+	"preferGlobal": true,
```

Comment:
This is supposed to be a boolean

Reply:
Great catch. Wish there was some IDE linting tool for package.json, but I'm yet to come across one